pmd-apex/src/test/resources/net/sourceforge/pmd/lang/apex/rule/performance/xml/OperationWithLimitsInLoop.xml

```diff
@@ -0,0 +1,413 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file now contains all test cases for the 4 cases that can trigger the rule. I copied them from the rules which are being deprecated.